doc/api/globals.md

```diff
@@ -583,6 +583,41 @@ The `MessagePort` class. See [`MessagePort`][] for more details.
 
 This variable may appear to be global but is not. See [`module`][].
 
+## `Navigator`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+An implementation of the [Navigator API][].
```

Comment:
```suggestion
A patrial implementation of the [Navigator API][].
```
should we mention this is not a full implementation?

Reply:
I’m -0 on this, but since you didn’t block the PR, I recommend merging this PR, if you’re ok with it @MoLow